bin/diagnostic.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== 5) {
+  answer = ask("Guess my secret? ");
+ if (answer < 1) {
+   console.log("Your guess has to be a number between 1-10.")
+ }
+ if(answer > 10) {
+   console.log("your guess has to be a number between 1-10.")
+ }
+ if (answer === 5) {
+  console.log("You guessed it!");
+  break;
+}
+}
```

Comment:
Good, but you could shorten the if statement by doing 
```js
if (answer < 1 || answer > 10) {
  console.log("Your guess has to be a number between 1-10.");
}
```

That is saying if the answer is less than 1 OR greater than 10